Sources/SmithyHTTPAPI/HTTPRequest.swift

```diff
@@ -274,3 +274,5 @@ extension HTTPRequestBuilder {
         return String(signatureSequence)
     }
 }
+
+extension HTTPRequest: @unchecked Sendable {}
```

Comment:
Rather than bury this extension at the bottom, make sure that it's clear at the point that the type is defined that it's unchecked and the developer must ensure the type is Sendable, not the compiler